### DISCUSSION TITLE
Is Netflix Conductor FIPS compliant?

### DISCUSSION THREAD

**Discussion by Ronomonaldo** [Netflix/conductor#3687] [Q&A]
Hi, I am a new to Netflix Conductor. My team and I are trying to find an orchestration tool that is FIPS compliant that we can integrate with our Spring Java application. Is Netflix Conductor currently FIPS compliant? If not, are there plans to ensure it is FIPS compliant in the near future?

Thanks!

**Comment by v1r3n** [comment]
Hi @Ronomonaldo open source has not gone through any compliance check for FIPS.  At [Orkes](https://orkes.io/) we run enterprise edition that is SOC2 compliant and goes through security and vulnerability checks regularly - while not certified to be FIPS compliant yet happy to work with you if you are interested.  We can chat [here](https://join.slack.com/t/orkes-conductor/shared_invite/zt-xyxqyseb-YZ3hwwAgHJH97bsrYRnSZg).
